dog.js

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
This works but is not good code design since you have a unique function for each element.  Can you think of ways to modify this to have one function that accomplishes the same task? You could consider passing additional parameters in your function call in the html. Or you could create some key value pairs to store the message and price for each dog and use the name to locate and display. We will get more into these techniques in other classes!